src/Raven.Client/Documents/Operations/CompareExchange/DeleteCompareExchangeValueOperation.cs

```diff
@@ -7,11 +7,24 @@
 
 namespace Raven.Client.Documents.Operations.CompareExchange
 {
+    /// <summary>
+    /// Operation to delete a compare exchange value in a RavenDB database. 
+    /// A compare exchange value is a key-value pair that is part of the RavenDB distributed coordination mechanism.
+    /// </summary>
+    /// <typeparam name="T">The type of the value associated with the compare exchange key.</typeparam>
     public sealed class DeleteCompareExchangeValueOperation<T> : IOperation<CompareExchangeResult<T>>
     {
         private readonly string _key;
         private readonly long _index;
 
+        /// <summary>
+        /// Operation to delete a compare exchange value in a RavenDB database. 
+        /// A compare exchange value is a key-value pair that is part of the RavenDB distributed coordination mechanism.
+        /// Initializes a new instance of the <see cref="DeleteCompareExchangeValueOperation{T}"/> class.
+        /// </summary>
+        /// <typeparam name="T">The type of the value associated with the compare exchange key.</typeparam>
+        /// <param name="key">The key of the compare exchange value to delete.</param>
+        /// <param name="index">The index of the compare exchange value to delete.</param>
```

Comment:
mention that the index is used for concurrency check, only if it match the operation will succeed

Reply:
Done